linked_list/main.py

```diff
@@ -62,34 +68,166 @@ class LinkedList(AbstractLinkedList):
     """
 
     def __init__(self, elements=None):
-        pass
+        self.elements = []      
```

Comment:
I understand the inclination to use a list to hold all the Nodes in a container. But since each Node has a pointer to the next Node in the list, you don't actually need a container to hold them all. You can traverse the list just by hopping from one Node to the next.